java/client/src/org/openqa/selenium/remote/RemoteWebDriver.java

```diff
@@ -322,6 +325,14 @@ public String getCurrentUrl() {
     }
   }
 
+  @Override
+  public Pdf print(Object printOptions) throws WebDriverException {
+    Response response = execute(DriverCommand.PRINT_PAGE((PrintOptions) printOptions));
+
+    Object result = response.getValue();
```

Comment:
The rest of the methods do a cast and assume the type is right. You can do `return new Pdf((String) result);`